docs/pages/includes/database-access/aws-bootstrap.mdx

```diff
@@ -64,6 +64,3 @@ You will need to run the bootstrap command once with the Teleport Database
 Service's IAM identity as the policy attachment target, and once for each AWS
 IAM role that is used for `assume_role_arn`.
 </Details>
-
-See the full `bootstrap` command
-[reference](../../database-access/reference/cli.mdx#teleport-db-configure-bootstrap).
```

Comment:
Why remove a link to the reference doc?

Reply:
This is to abide by our style guide rule to not include links in the body text unless they're necessary. It's not necessary to follow the reference guide to get through the how-to guides that include this partial.